pkg/universal-cnf/vppagent/backend.go

```diff
@@ -98,6 +98,23 @@ func (b *UniversalCNFVPPAgentBackend) ProcessClient(
 	return nil
 }
 
+func (b *UniversalCNFVPPAgentBackend) buildVppIfName(defaultIfName, serviceName string, conn *connection.Connection) string {
+	// NSC peer connection
+	if name, ok := conn.Labels["podName"]; ok {
```

Comment:
not sure if there's a networkservicemesh/networkservicemesh constant for this that we can use

Reply:
Looks like it's in: `networkservicemesh/networkservicemesh/controlplane/api/connection/constants.go`